qube/create/templates/lib/lib_groovy/{{ cookiecutter.project_slug }}/README.rst

```diff
@@ -0,0 +1,44 @@
+{{ cookiecutter.project_name }}
+-----------------------------------
+
+.. image:: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/Build%20Maven%20Package/badge.svg
+    :target: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/Build%20Maven%20Package/badge.svg
+    :alt: Github Workflow Build Maven Package Status
+
+.. image:: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/Run%20Maven%20Tests/badge.svg
+    :target: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/Run%20Maven%20Tests/badge.svg
+    :alt: Github Workflow Tests Status
+
+.. image:: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/QUBE%20lint/badge.svg
+    :target: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/QUBE%20lint/badge.svg
+    :alt: QUBE Lint Status
+
+.. image:: https://img.shields.io/travis/qbicsoftware/{{ cookiecutter.project_slug }}.svg
```

Comment:
get rid of travis

there is no travis anymore

should also be fixed in all templates

Reply:
done and mentioned in issue